go/vt/wrangler/reparent.go

```diff
@@ -413,8 +413,8 @@ func (wr *Wrangler) plannedReparentShardLocked(ctx context.Context, ev *events.R
 	}
 	ev.OldMaster = *oldMasterTabletInfo.Tablet
 
-	// create a new context for the short running remote operations
-	remoteCtx, remoteCancel := context.WithTimeout(ctx, *topo.RemoteOperationTimeout)
+	// create a new context for the remote operations associated with promoting the new master
+	remoteCtx, remoteCancel := context.WithTimeout(ctx, waitSlaveTimeout)
```

Comment:
DemoteMaster and UndoDemoteMaster should not be using waitSlaveTimeout.
Also, default waitSlaveTimeout to topo.RemoteOperationTimeout at https://github.com/vitessio/vitess/blob/master/go/vt/vtctl/reparent.go#L103

Reply:
thanks, i've updated.